### DISCUSSION TITLE
Directly flash USB drive

### DISCUSSION THREAD

**Discussion by apbr** [OE4T/meta-tegra#1498] [Q&A]
With the Nvidia scripts it seems to be possible to directly flash a bootable USB drive that is plugged into the host device without using any Jetson device. (See [To set up a flash drive manually for booting](https://docs.nvidia.com/jetson/archives/r35.4.1/DeveloperGuide/text/SD/FlashingSupport.html#to-set-up-a-flash-drive-manually-for-booting))
Is this already possible for meta-tegra based distros?
If not do you have any hints where to start with for adding such feature?

**Comment by madisongh** [comment]
We don't have anything already built in, but it should be possible to adapt the `initrd-flash` script, or even the `dosdcard.sh` script, to do this.  Specifics on whether it will work or not may depend on the target Jetson hardware, though.
